### PR TITLE
Recipe for synthpop

### PR DESCRIPTION
**Pull request by smmaurer**
Adding a recipe for SynthPop: https://github.com/UDST/synthpop

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by smmaurer** [comment]
@conda-forge/help-python i think this is ready for review!

File: recipes/synthpop/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2016, UrbanSim Inc. All rights reserved.
```
Comment (isuruf) [inline recipes/synthpop/LICENSE.txt]: Can you open an issue upstream about the license file missing in sdist?

Comment (smmaurer) [reply recipes/synthpop/LICENSE.txt]: Good idea. I just added a MANIFEST file, which should take care of it when the next release comes out: https://github.com/UDST/synthpop/commit/f4800870291490f38af2e4b2be73222c5c24d353

File: recipes/synthpop/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - census>=0.5
```
Comment (chrisburr) [inline recipes/synthpop/meta.yaml]: You need a single space here or the feedstock creation will fail
```suggestion
    - census >=0.5
```
(same for the others)

Comment (smmaurer) [reply recipes/synthpop/meta.yaml]: Ooh, thanks for catching this! Just fixed it

**Pull request merged**